Add sunset disclaimer to self-host overview (#6023)
PostHog/posthog.com: contents/docs/self-host/index.mdx

---

import Disclaimer from './\_snippets/disclaimer.mdx'
import Sunset from '../self-host/\_snippets/sunset-disclaimer.mdx'

> > **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We are currently in the process of [sunsetting our Kubernetes deployment](/blog/sunsetting-helm-support-posthog).

Reviewer: ```suggestion
> **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We are currently in the process of [sunsetting our paid Kubernetes deployment](/blog/sunsetting-helm-support-posthog).
```

> Our Docker compose deployment is a great way to test some features, or to deploy for hobby projects which don't need advanced tools. You can spin up a fresh PostHog instance in a few minutes - though [PostHog Cloud](/pricing) is cheaper and easier to use for teams and professional products. 

Author: ```suggestion
Our MIT-licensed Docker compose deployment is a great for internal tools, or evaluating features without vendor approval. It's also great for hobby projects which don't need advanced tools. You can spin up a fresh PostHog instance in a few minutes - though [PostHog Cloud](/pricing) is cheaper and easier to use for teams and professional products. 
```

> Our MIT-licensed Docker compose deployment is a great for internal tools, or evaluating features without vendor approval. It's also great for hobby projects which don't need advanced tools. You can spin up a fresh PostHog instance in a few minutes - though [PostHog Cloud](/pricing) is cheaper and easier to use for teams and professional products. 

Reviewer 2: I'd probably just remove the last sentence, since it's already mentioned in the next paragraph - otherwise it feels superlative. Other than that, lgtm!

Author: ```suggestion
Our MIT-licensed Docker compose deployment is a great for internal tools, or evaluating features without vendor approval. It's also great for hobby projects which don't need advanced tools. 
```

> Want more reliability? The easiest way to get started with PostHog is to use [PostHog Cloud](https://app.posthog.com/signup).

Reviewer: ```suggestion
Want no hosting cost or hassle? The easiest way to get started with PostHog is to use [PostHog Cloud](https://app.posthog.com/signup), which comes with a big free tier.
```

> Want no hosting cost or hassle? The easiest way to get started with PostHog is to use [PostHog Cloud](https://app.posthog.com/signup), which comes with a big free tier.

Reviewer 3: ```suggestion
Want no hosting cost or hassle? The easiest way to get started with PostHog is to use [PostHog Cloud](https://app.posthog.com/signup), which comes with a generous free tier.
```
